data/org.freedesktop.impl.portal.GlobalShortcuts.xml

```diff
@@ -106,6 +106,28 @@
       <arg type="a{sv}" name="results" direction="out"/>
     </method>
 
+    <!--
+        ConfigureShortcuts:
+        @session_handle: Object path for the :ref:`org.freedesktop.portal.Session` object representing the session
+        @parent_window: Identifier for the application window, see :doc:`window-identifiers`
+        @options: Vardict with optional further information
+
+        Request showing a configuration UI so the user is able to conigure all shortcuts of this session.
+
+        Supported keys in the @options vardict include:
+
+        * ``activation_token`` (``s``)
+
+          A token that can be used to activate the chosen application.
```

Comment:
I know you just copied this, like we did all over the place, but there is no "chosen application". Maybe something like "activate the configuration window".